### ISSUE TITLE
Replace deprecated image

### ISSUE THREAD

**Issue by asfaltboy** [jfrog/artifactory-orb#16]
### Orb version

1.0.1

### What happened

We use `artifactory-orb/upload` job, and CircleCI warns about using a deprecated image.

![Screenshot 2022-08-22 at 12 59 59](https://user-images.githubusercontent.com/321520/185916391-47682323-a62f-44cc-94c2-eb65b52fd49c.png)

References:
* https://discuss.circleci.com/t/legacy-convenience-image-deprecation/41034
* https://circleci.com/blog/announcing-our-next-generation-convenience-images-smaller-faster-more-deterministic/
* https://circleci.com/docs/next-gen-migration-guide
* https://github.com/CircleCI-Public/cimg-base

### Expected behavior

tl;dr I think we can switch to any of the new CircleCI images, e.g `cimg/base`, as we don't use the JDK or Java.

It doesn't seem we're building the tool from source, and the shell script used [here](https://github.com/jfrog/artifactory-orb/blob/master/src/commands/install.yml#L13-L27) (i.e https://getcli.jfrog.io , [see source](https://github.com/jfrog/jfrog-cli/blob/cd7ef8715dc9468c288bc5e67095bb33e1995395/build/getcli/jfrog.sh)) does a good job at finding the correct binary distribution to use based on system architecture.
